From 1095e927ae6a5ce730d696d734c771e4dfa07337 Mon Sep 17 00:00:00 2001
From: Auke Kok <auke-jan.h.kok@intel.com>
Date: Thu, 16 Feb 2017 11:25:37 -0800
Subject: [PATCH] Better file locations.

---
 configure.ac    |  4 ++--
 doc/Makefile.am | 29 ++---------------------------
 2 files changed, 4 insertions(+), 29 deletions(-)

diff --git a/configure.ac b/configure.ac
index fe577db..b0e09db 100644
--- a/configure.ac
+++ b/configure.ac
@@ -354,8 +354,8 @@ AC_OUTPUT_COMMANDS([
 
 #define SIEGE_HOME     "$prefix"
 #define URL_FILE       "$sysconfdir/urls.txt"
-#define CNF_FILE       "$sysconfdir/siegerc"
-#define LOG_FILE       "$localstatedir/siege.log"
+#define CNF_FILE       "/usr/share/defaults/siege/siegerc"
+#define LOG_FILE       "/var/log/siege.log"
 
 
 #endif/*JOEPATH_H*/
diff --git a/doc/Makefile.am b/doc/Makefile.am
index c9adc84..61d06ee 100644
--- a/doc/Makefile.am
+++ b/doc/Makefile.am
@@ -42,37 +42,12 @@ siegerc.in
 
 DISTCLEANFILES    =      siegerc $(man_MANS)
 
-SIEGERC           =      $(sysconfdir)/siegerc
+SIEGERC           =      /usr/share/defaults/siege/siegerc
 
 URLSTXT           =      $(sysconfdir)/urls.txt
 
 install-exec-hook: 
-	@if test -f $(DESTDIR)$(SIEGERC); then \
-	  if cmp -s $(srcdir)/siegerc $(DESTDIR)$(SIEGERC); then echo ""; \
-	    else \
-	    echo ' $(INSTALL_DATA) $(srcdir)/siegerc $(DESTDIR)$(SIEGERC).new'; \
-	    $(INSTALL_DATA) $(srcdir)/siegerc $(DESTDIR)$(SIEGERC).new; \
-	    echo "#####################################################"; \
-	    echo "WARNING: File $(SIEGERC) already exists."; \
-	    echo "         A new resource file has been installed as"; \
-	    echo "         $(SIEGERC).new. You may want to"; \
-	    echo "         consider using the newer version in order to"; \
-	    echo "         take advantage of any new features."; \
-	    echo "#####################################################"; \
-	  fi; \
-	else \
-	  $(INSTALL_DATA) $(srcdir)/siegerc $(DESTDIR)$(SIEGERC); \
-	fi 
-	@if test -f $(DESTDIR)$(URLSTXT); then \
-	  if cmp -s $(srcdir)/siegerc $(DESTDIR)$(URLSTXT); then echo ""; \
-	    else \
-	    echo "WARNING: File $(DESTDIR)$(URLSTXT) already exists."; \
-	    echo "         It was NOT replaced with this installation."; \
-	  fi; \
-	else \
-	  $(mkinstalldirs) $(DESTDIR)$(sysconfdir); \
-	  $(INSTALL_DATA) $(srcdir)/urls.txt $(DESTDIR)$(URLSTXT); \
-	fi 
+	$(INSTALL_DATA) -D $(srcdir)/siegerc $(DESTDIR)$(SIEGERC);
 
 
 uninstall:
-- 
2.11.1

